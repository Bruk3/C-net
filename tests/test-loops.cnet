//testing loops 

for(count = 1; count <= num; count= COUNT+1)
{
    sum += count;
}