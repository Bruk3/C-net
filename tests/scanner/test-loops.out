FOR 
LPAREN
ID(count)
ASSIGN 
INTLIT(1)
SEMI 
ID(count)
LEQ
ID(num)
SEMI 
ID(count)
ASSIGN 
ID(COUNT)
PLUS
INTLIT(1)
RPAREN 
LBRACE
ID(sum)
PLUSEQ
ID(count)
SEMI 
RBRACE
EOF